src/NetAnalyzers/Core/Microsoft.NetCore.Analyzers/MicrosoftNetCoreAnalyzersResources.resx

```diff
@@ -1698,4 +1698,13 @@
   <data name="StaticAndAbstractRequiresPreviewFeatures" xml:space="preserve">
     <value>Using both 'static' and 'abstract' modifiers requires opting into preview features. See https://aka.ms/dotnet-warnings/preview-features for more information.</value>
   </data>
+  <data name="CallAsyncMethodInAsyncContextMessage" xml:space="preserve">
+    <value>'{0}' synchronously blocks. Await '{1}' instead.</value>
+  </data>
+  <data name="CallAsyncMethodInAsyncContextTitle" xml:space="preserve">
+    <value>Call async methods when in an async method</value>
```

Comment:
Is this how it is worded in the VS THreading repo?

Reply:
Yes that's what they had